src/ring_theory/polynomial/chebyshev/defs.lean

```diff
@@ -4,32 +4,48 @@ Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Johan Commelin
 -/
 
-import data.polynomial.eval
+import data.polynomial.derivative
+import tactic.ring
 
 /-!
 # Chebyshev polynomials
 
 The Chebyshev polynomials are two families of polynomials indexed by `ℕ`,
 with integral coefficients.
-In this file, we only consider Chebyshev polynomials of the first kind.
 
 See the file `ring_theory.polynomial.chebyshev.basic` for more properties.
 
-## Main declarations
+## Main definitions
 
 * `polynomial.chebyshev₁`: the Chebyshev polynomials of the first kind.
+* `polynomial.chebyshev₂`: the Chebyshev polynomials of the second kind.
 * `polynomial.lambdashev`: a variant on the Chebyshev polynomials that define a Lambda structure
   on `polynomial ℤ`.
 
+## Main statements
+
+* The formal derivative of the Chebyshev polynomials of the first kind is a scalar multiple of the
+  Chebyshev polynomials of the second kind.
+
 ## Implementation details
 
 In this file we only give definitions and some very elementary simp-lemmas.
 This way, we can import this file in `analysis.special_functions.trigonometric`,
 and import that file in turn, in `ring_theory.polynomial.chebyshev.basic`.
 
+The basic API is taken from https://arxiv.org/pdf/2008.03575.pdf
```

Comment:
Can you add this reference to `docs/references.bib` and cite that entry here?

Reply:
ooh, you're right, that's better.